### DISCUSSION TITLE
WhatsApp Icon required

### DISCUSSION THREAD

**Discussion by ShaunHalliday** [twilio-labs/paste#3944] [Request: Icon] [answered]
### Description

Hi,

I am rebuilding one of our Flex plugins as we are moving from Flex 1 to Flex 2 and we need to replace MUI icons with Paste icons. I have reviewed the Paste icon set and cannot find a WhatsApp icon, please could this be added, information as per guidelines below.

1 - Link to icon - https://www.streamlinehq.com/icons/font-awesome-solid?search=whatsapp&icon=ico_cWinNmZodeN1iHDo

2 - This will be used in Flex as a UI clickable icon, this will be used as both a sidenav link and also within UI to perform an action.

3 - This will be a UI icon please.

Kind Regards,

Shaun

### Link to icon

https://www.streamlinehq.com/icons/font-awesome-solid?search=whatsapp&icon=ico_cWinNmZodeN1iHDo

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by serifluous** [comment, marked as answer]
Hey @ShaunHalliday !

We unfortunately don't add 3rd party libraries to Paste. You can check out an explanation here: https://github.com/twilio-labs/paste/discussions/3544#discussioncomment-7268153

To add it to your Flex plugin, though, you can grab the [SVG from WhatsApp's brand website](https://about.meta.com/brand/resources/whatsapp/whatsapp-brand/) and use our [SVG-to-React library](https://paste.twilio.design/core/libraries#svg-to-react) to add convert it to a component. Hope that helps!
